scipy/stats/_continuous_distns.py

```diff
@@ -935,23 +935,28 @@ def asymptotic_b_large(a, b):
             log_term = sum_ab*np.log1p(a/b) + np.log(b) - 2*np.log(sum_ab)
             return t1 + t2 + log_term
 
-        def threshold_large(v):
-            if v == 1.0:
-                return 1000
+        def asymptotic_a_large(a, b):
+            return asymptotic_b_large(b, a)
 
+        def threshold_large(v):
             j = np.log10(v)
-            digits = int(j)
-            d = int(v / 10 ** digits) + 2
-            return d*10**(7 + j)
```

Comment:
Instead of casting to int we could do `np.floor`. Not sure why the casting to int was necessary in the first place.